### ISSUE TITLE
TGD should use default Terminal.Gui look & feel

### ISSUE THREAD

**Issue by tig** [gui-cs/TerminalGuiDesigner#291]
I love that TGD demonstrates some cool UI flair, like the 3D button effect... some project should be pushing the envelope there.

However, I don't think TGD should be that project. IMO, TGD 

- should be a showcase for the default TG look & feel
- any look & feel that the TGD maintainers really think is needed to make TGD better should be treated as high-priority feature requests for TG v2

List of current things I see TGD doing that are counter to the built-in v2 look & feel

- Button shadows
- Centered dialog titles
- Different line styles in dialogs
- Quit key is CTRL-Q not Esc

**Comment by tznind** [comment]
I agree this makes sense, although I did have fun making the drop shadows.

The one thing that I do not want to loose is the start screen which was modelled after visual studio codes Start screen with just the most likely commands on it.

![image](https://github.com/gui-cs/TerminalGuiDesigner/assets/31306100/f20934ce-2504-4ab3-9af6-b51064a696c7)
